web/themes/contrib/civictheme/includes/mobile_navigation.inc

```diff
@@ -46,46 +37,44 @@ function _civictheme_preprocess_block__civictheme_mobile_navigation(array &$vari
   foreach ($menu_fields as $menu_key => $menu_field) {
     $menu_block = civictheme_get_field_referenced_entity($entity, $menu_field);
 
-    if (!$menu_block instanceof BlockInterface) {
+    if (!$menu_block instanceof BlockInterface || !str_starts_with($menu_block->getPluginId(), 'menu_block:')) {
```

Comment:
Can we use MenuBlock instance check?

Reply:
hi @richardgaunt I tried to use MenuBlock here but it doesn't work so I will keep it for now.